test-tests/shared/src/main/scala/zio/test/TimeVariantsSpec.scala

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2019 John A. De Goes and the ZIO Contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package zio.test
+
+import java.time.{ Instant, LocalDateTime, OffsetDateTime, ZoneOffset }
+
+import scala.concurrent.Future
+
+import zio.duration._
+import zio.test.GenUtils.{ checkSample, checkShrink }
+import zio.test.TestUtils.label
+
+object TimeVariantsSpec extends AsyncBaseSpec {
+
+  val run: List[Async[(Boolean, String)]] = List(
+    label(anyFiniteDurationShrinksToZero, "anyFiniteDuration shrinks to zero"),
```

Comment:
Can we add properties about what any of the other generators shrink to?

Reply:
Thank you for your feedback! I'm still working on improvements...

Reply:
No worries. Just reach out whenever you're ready for another review.